src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/OrganisationalInternalControllerProviderTest.java

```diff
@@ -197,7 +224,7 @@ private Organisation getOrganisation() {
         contactInformation.setAddressLine2("addressLine2");
         contactInformation.setCountry("country");
         contactInformation.setPostCode("HA5 1BJ");
-        organisation.setContactInformations(Arrays.asList(contactInformation));
+        organisation.setContactInformations(asList(contactInformation));
```

Comment:
Please use `List.of()` to return a list instead of `Arrays.asList()`

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1096/commits/b17549eb7a6a39bc04a0482a299739fe4dc1e212